### ISSUE TITLE
VoiceOver on iOS

### ISSUE THREAD

**Issue by NajaWulff** [Notalib/LYT#118]
There might be several issues hiding here. First of all, a more complete test of whether the player actually works with Voiceover should be made. Naja is assigned

Right, so this issue can be dealt with in various ways - I suggest we start to improve various voiceover issues by using the WAI-ARIA standard

http://juicystudio.com/article/examining-wai-aria-properties.php

When loggin on, or on suspended loading times, VoiceOver doesn't register the 'Loggin on' or 'Loading' popup messages.
